sql/2019/05_Third_Parties/05_03.sql

```diff
@@ -10,12 +10,17 @@ FROM (
   SELECT
       client,
       type AS contentType,
-      ThirdPartyTable.category AS thirdPartyCategory
+      IFNULL(ThirdPartyTable.category,
+        IF(DomainsOver50Table.requestDomain IS NULL, 'first-party', 'other')
```

Comment:
Why are requests in which the domain is unknown (null) treated as first-party? Wouldn't this be the type of request you'd put in the other category?

Reply:
`DomainsOver50Table` contains a list of the all the third-party domains. If the request domain is not a third-party domain, it's assumed to be first-party.